go.mod

```diff
@@ -3,10 +3,10 @@ module github.com/Venafi/vault-pki-backend-venafi
 go 1.21
 
 require (
-	github.com/Venafi/vcert/v5 v5.1.1
+	github.com/Venafi/vcert/v5 v5.4.1-0.20240228184021-dd6dc944a40f
```

Comment:
Make sure this is pointing to the correct released version and not a commit

Reply:
Totally